README.md

```diff
@@ -21,39 +20,24 @@ This project contains Ansible code that creates a baseline in an existing kubern
   - kustomize such that data and homes directories are mounted on cas nodes and on compute server instances
   - Deploy [Viya Monitoring for Kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes)
   - Deploy MPP or SMP CAS servers
+
 - Manage Viya Deployments
   - Organize and persist config for any number of Viya deployments across namespaces, cluster, or cloud providers.
 
-### Prerequisites
-
-This tool supports running both from ansible installed on your local machine or via a docker container. The Dockerfile for the container can be found [here](Dockerfile)
+## Prerequisites
 
-#### Ansible
+### Operational knowledge of 
 
-- terraform
-- ansible
-- unzip
-- tar
-- kubectl
-- kustomize
-- python
-- pip
-- openshift pip module
-- kubernetes  pip module
-- dnspython pip module
-- community.kubernetes ansible-galaxy module
-- ansible.posix ansible-galaxy module
-- helm
-- git
+- [Ansible](https://docs.ansible.com/ansible/latest/user_guide/index.html#getting-started)
+- [Docker](https://www.docker.com/)
+- [Kubernetes](https://kubernetes.io/docs/concepts/)
+- Cloud Provider
 
-#### Docker
+### Technical
 
-- docker
-- git
+- [Ansible and docker dependencies](docs/user/Dependencies.md)
 
-note: For dependency versions and changing versions see [Dependency Versions](docs/DEPENDENCY-VERSIONS.md)
-
-#### Infrastructure
+### Infrastructure
 
 Prior to running this playbook some infrastructure needs to be in place
 
```

Comment:
Reorder some wording under storage. Let's immediately tell them that Viya 4 IAC projects will set this up for them, so customers don't have to ready this whole section unless needed. Also capitalize NFS (industry standard)

"Storage: When using NFS based storage (like Azure NetApp or EFS), the storage needs certain folders setup. The Viya 4 IAC projects automate the storage setup if desired. "

"NFS: There needs to be a PVs folder created under the export path. This is used for persistent volumes. Additionally, a folder needs to be created for each cluster with sub-folders for bin, data, and homes. Below is the required NFS exports folder structure"

"JumpBox: This tool can manage NFS folders if you provide ssh access to a JumpBox that has the nfs storage mounted to it at <JUMP_SVR_RWX_FILESTORE_PATH>. If you wish to manage the nfs server yourself, the JumpBox is not required. Below is the JumpBox folder structure"

Reply:
I don't believe we should add the suggested items. Let's have a conversation about that.

Reply:
After talking with @hahewlet She is going to rework the wording and touch base with @NormJohnIV .

Reply:
Storage 
A file server that uses the network file system (NFS) protocol is the minimum requirement. You can either use one of the Viya 4 IAC projects to create the required storage or bring your own K8s storage.


If the Viya 4 IAC projects were used to create an NFS server VM and JumpBox/Basitan server VM, the information can be passed in to viya4-deployment and the required directory structure below will be created. 


If bringing your own a server that uses NFS, the following NFS exports folder structure must be created prior to running viya4-deployment. The JumpBox/Bastian can manage NFS folders if you provide ssh access to a JumpBox that has the nfs storage mounted to it at <JUMP_SVR_RWX_FILESTORE_PATH>. If you wish to manage the nfs server yourself, the JumpBox is not required. Below is the JumpBox folder structure.

NFS:
```bash
  <export_dir>        <- nfs export path
  <export_dir>        <- NFS export path
    /pvs              <- location for persistent volumes
    /<namespace>      <- folder per namespace
      /bin            <- location for open source directories
      /data           <- location for SAS and CAS Data
      /homes          <- location for user home directories to be mounted
      /astores        <- location for astores
  ```
JumpBox/Basian:
 ```bash
  <JUMP_SVR_RWX_FILESTORE_PATH>     <- mounted NFS export
    /pvs                            <- location for persistent volumes
    /<namespace>                    <- folder per namespace
      /bin                          <- location for open source directories
      /data                         <- location for SAS and CAS Data
      /homes                        <- location for user home directories to be mounted
      /astores                      <- location for astores
  ```

Reply:
updated